contracts/its-stacks-translator/src/contract/its_clarity_bytes_to_hub_message.rs

```diff
@@ -0,0 +1,422 @@
+use std::str::FromStr;
+
+use clarity::vm::representations::ClarityName;
+use clarity::vm::types::{
+    BufferLength, SequenceSubtype, StringSubtype, TupleData, TupleTypeSignature, TypeSignature,
+    Value,
+};
+use cosmwasm_std::Uint128;
+use cw_storage_plus::KeyDeserialize;
+use interchain_token_service_std::{
+    DeployInterchainToken, HubMessage, InterchainTransfer, Message, TokenId,
+};
+use router_api::ChainNameRaw;
+use stacks_common::codec::StacksMessageCodec;
+
+use crate::contract::its_hub_message_to_clarity_bytes::{
+    CLARITY_NAME_AMOUNT, CLARITY_NAME_DATA, CLARITY_NAME_DECIMALS,
+    CLARITY_NAME_DESTINATION_ADDRESS, CLARITY_NAME_NAME, CLARITY_NAME_SOURCE_ADDRESS,
+    CLARITY_NAME_SYMBOL, CLARITY_NAME_TOKEN_ID, CLARITY_NAME_TYPE,
+    MESSAGE_TYPE_DEPLOY_INTERCHAIN_TOKEN, MESSAGE_TYPE_INTERCHAIN_TRANSFER,
+};
+use crate::error::ContractError;
+
+const MESSAGE_TYPE_SEND_TO_HUB: u128 = 3;
+
+const CLARITY_NAME_DESTINATION_CHAIN: &str = "destination-chain";
+const CLARITY_NAME_PAYLOAD: &str = "payload";
+const CLARITY_NAME_MINTER: &str = "minter";
+const CLARITY_SIZE_DESTINATION_CHAIN: u32 = 19;
+const CLARITY_SIZE_PAYLOAD: u32 = 63_000;
+const CLARITY_SIZE_TOKEN_ID: u32 = 32;
+const CLARITY_SIZE_DESTINATION_ADDRESS: u32 = 128;
```

Comment:
some of these constants are defined multiple times in different places. It might make sense to just move them to the ampd folder and include them elsewhere?

Reply:
Extracted all constants to a simple crate under `packages/stacks-types`.